controller/controller_manager.go

```diff
@@ -75,7 +75,7 @@ func StartControllers(logger logrus.FieldLogger, stopCh chan struct{}, controlle
 
 	ds := datastore.NewDataStore(lhInformerFactory, lhClient, kubeInformerFactory, kubeClient, namespace)
 
-	proxyHandler := engineapi.NewEngineClientProxyHandler(logger, ds)
+	proxyHandler := engineapi.NewEngineClientProxyHandler(logger, ds, time.Second*5, stopCh)
```

Comment:
Can we rename to `engineClientProxyHandler`? more specific.

Reply:
Updated.